ui/pages/confirmations/components/confirm/info/approve/edit-spending-cap-modal/edit-spending-cap-modal.tsx

```diff
@@ -116,10 +128,17 @@ export const EditSpendingCapModal = ({
     setCustomSpendingCapInputValue(formattedSpendingCap.toString());
   }, [customSpendingCapInputValue, formattedSpendingCap]);
 
+  const showDecimalError = useMemo(() => {
+    return (
+      decimals &&
+      parseInt(decimals, 10) < countDecimalDigits(customSpendingCapInputValue)
+    );
+  }, [decimals, customSpendingCapInputValue]);
+
   return (
     <Modal
       isOpen={isOpenEditSpendingCapModal}
-      onClose={() => setIsOpenEditSpendingCapModal(false)}
+      onClose={() => handleCancel()}
```

Comment:
Minor, no need for the lambda since `handleCancel` is already a function reference with no arguments?